src/generateNumberCards.js

```diff
@@ -0,0 +1,25 @@
+function generateRandomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function generate3To5NumberCards(){
+let randomNumbersHolder = {}
```

Comment:
why is this an object instead of an array? this is making your code much more complicated

Reply:
Agreed, carried over thinking from a different approach that we abandoned.

changed to an array now